src/na/fomac/Device.cpp

```diff
@@ -116,10 +116,11 @@ auto FoMaC::Device::initDurationUnitFromDevice() -> bool {
   return true;
 }
 auto FoMaC::Device::initDecoherenceTimesFromDevice() -> bool {
-  // find first non-zone site as reference
-  auto regularSites = getSites() | std::views::filter([](const auto& site) {
-                        return !site.isZone().value_or(false);
-                      });
+  const auto regularSites = getQubits();
```

Comment:
```suggestion
  const auto regularSites = getRegularSites();
```
For consistency witth the notions used in the QDMI spec (as far as I recall) this method should probably be renamed. I must have overlooked this above.